.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,91 @@
+name: test-deploy
+
+on:
+  push:
+    branches:
+      [9675-mediasoup-config]
```

Comment:
guessing this script needs to be removed

Reply:
Yes, it's gone.